libs/wingc/src/type_check.rs

```diff
@@ -2283,19 +2349,62 @@ impl<'a> TypeChecker<'a> {
 						),
 						reassignable: false,
 						flight: Phase::Independent,
+						is_static: false,
 					},
 				};
 
 				if force_reassignable {
-					VariableInfo {
-						_type: res._type,
-						reassignable: true,
-						flight: res.flight,
-					}
+					let mut res = res.clone();
+					res.reassignable = true;
+					res
```

Comment:
Just for fun, here's a third way to do this (random rust trivia)
```suggestion
					VariableInfo {
					  reassignable: true
					  ..res.clone(),
					}
```

Reply:
😲!

Reply:
Stockholm syndrome or not, that's simply cool.